command/common.go

```diff
@@ -36,3 +52,117 @@ func ValidateMinMaxValidatorsNumber(minValidatorCount uint64, maxValidatorCount
 
 	return nil
 }
+
+// GetValidatorsFromPrefixPath extracts the addresses of the validators based on the directory
+// prefix. It scans the directories for validator private keys and compiles a list of addresses
+func GetValidatorsFromPrefixPath(
+	prefix string,
+	validatorType validators.ValidatorType,
+) (validators.Validators, error) {
+	files, err := ioutil.ReadDir(".")
+	if err != nil {
+		return nil, err
+	}
+
+	validatorSet := validators.NewValidatorSetFromType(validatorType)
+
+	for _, file := range files {
+		path := file.Name()
+
+		if !file.IsDir() {
```

Comment:
Nitpick: we can put these to conditions in one line 
	if !cond1 || !cond2 {
		continue
	}

Reply:
Fixed 811117d627f466099f4c681283f8fd44de32ab4c